server/models/Contest.js

```diff
@@ -0,0 +1,32 @@
+const { Schema, model } = require("mongoose");
+
+const contestSchema = new Schema({
+    title: {
+        type: String,
+        required: true,
+    },
+    description: {
+        type: String,
+        required: true,
+    },
+    prize_amount: {
```

Comment:
Just a suggestion, as MongoDB speaks JavaScript, try to utilize JS naming conventions of camelCase while defining field names.

Reply:
Thank You, will update schema with changes.